WordPress/WordPressTest/StatsTrafficTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+@testable import WordPress
+
+final class StatsTrafficTests: XCTestCase {
```

Comment:
nit-pick but I think we can have a separate test file for this picker

```suggestion
final class StatsTrafficDatePickerViewModel: XCTestCase {
```